middleware/2023-09-06-1.2.21103.40594.md

```diff
@@ -0,0 +1,29 @@
+---
+authors: poscreator
+slug: middleware/1.2.21103.40594
+tags: [Middleware, France]
+---
+
+# Middleware 1.2.21103.40594 SignaturCloud (France)
```

Comment:
```suggestion
# Middleware 1.161.6.6274 SignaturCloud (France)
```

FYI you can get the current version via this link https://signaturcloud.fiskaltrust.fr/api/version